CONTRIBUTING.md

```diff
@@ -9,10 +9,12 @@ Of course, contributing code is more than welcome! If you're planning to submit
 
 If you are interested in contributing documentation, please note the following:
 
-
 - Doc issues are labeled with the `doc` label.
 - The thin-edge.io docs content is in the `docs/src/` directory.
 
+[How to build from source.](./BUILDING.md)
```

Comment:
I think we want to have discussion where to give the link to the doc how to build. For me, we can put it in README, where everyone can see it for the first.

Reply:
It wasn't specified anywhere where the link should go, I just chose a place I felt was good enough. As our README has nothing like 'quick start' or 'how to' it just didn't feel right to place it there as our main intention is for users to use as packages from releases and not build it from source.
Maybe Contributing section of readme would be good.

Reply:
As Sebastian said, putting the links in both in README and CONTRIBUTING make sense. I agree that contributing section of readme is a good place.